en_ewt-ud-dev.conllu

```diff
@@ -28744,13 +28744,13 @@
 # text = I highly recommend Bay View if you are looking for Accommodation in Camps Bay.
 1	I	I	PRON	PRP	Case=Nom|Number=Sing|Person=1|PronType=Prs	3	nsubj	3:nsubj	_
 2	highly	highly	ADV	RB	_	3	advmod	3:advmod	_
-3	recommend	recommend	VERB	VBP	Mood=Ind|Number=Sing|Person=1|Tense=Pres|VerbForm=Fin	0	root	0:root	_
+3	recommend	recommend	VERB	VBP	Mood=Ind|Number=Sing|Person=1|Tense=Pres|VerbForm=Fin	0	root	0:root	Cxn=Conditional-UnspecifiedEpistemic-NoInversion|CxnElt=3:Conditional-UnspecifiedEpistemic-NoInversion.Apodosis
 4	Bay	Bay	PROPN	NNP	Number=Sing	5	compound	5:compound	_
 5	View	View	PROPN	NNP	Number=Sing	3	obj	3:obj	_
 6	if	if	SCONJ	IN	_	9	mark	9:mark	_
 7	you	you	PRON	PRP	Case=Nom|Person=2|PronType=Prs	9	nsubj	9:nsubj	_
 8	are	be	AUX	VBP	Mood=Ind|Number=Sing|Person=2|Tense=Pres|VerbForm=Fin	9	aux	9:aux	_
-9	looking	look	VERB	VBG	Tense=Pres|VerbForm=Part	3	advcl	3:advcl:if	_
+9	looking	look	VERB	VBG	Tense=Pres|VerbForm=Part	3	advcl	3:advcl:if	CxnElt=3:Conditional-UnspecifiedEpistemic-NoInversion.Protasis
```

Comment:
> Happened to notice that this one is a speech act conditional. I don't think we have a specific label for those.

Hi, @nschneid !
I guess it wouldn't be right to tag «if» in features as a Protasis marker, but it would be helpful for distinguishing the «conditional» marking in Mordvin languages, where the verbs take a protasis morpheme and called «Conditional» in traditional grammars.
Would you have any suggestions for a «Protasis» feature?

Reply:
@rueter The UCxn approach implemented here is to identify the content elements of the construction via Grew rules that incorporate grammatical marking. We don't tag the functional morphemes like "if" directly. UD morphological features (at least as I understand them) are generally for indicating where a word form lies in a paradigm, not for what an uninflecting function word like "if" contributes to an analytic construction. Does that answer your question?